static/src/javascripts/lib/ajax.js

```diff
@@ -13,7 +12,21 @@ const ajax = (params) => {
         options.crossOrigin = true;
     }
 
-    const r = reqwest(options);
+    const { url } = options;
+    const headers = { ...options.headers };
+    if(options.headers !== undefined) headers.append( options.headers );
```

Comment:
`append` isn't a function on arrays in javascript, only on `ParentNode`. This will probably need changing to 
`if(options.headers !== undefined) headers.push( options.headers );`

Reply:
Should add that this seemed to stop comments loading for signed in users on live blogs/non-DCR articles

Reply:
Absolutely, please see: #23629